skype/skype-ps/skype/grant-CsTeamsCallHoldPolicy.md

```diff
@@ -0,0 +1,227 @@
+2) the user's user principal name (UPN);
+3) the user's Active Directory display name (for example, Ken Myer).
+
+```yaml
+Type: UserIdParameter
+Parameter Sets: Identity
+Aliases:
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -PassThru
+Enables you to pass a user object through the pipeline that represents the user account being assigned the Teams call hold policy.
+By default, the Grant-CsTeamsCallHoldPoly cmdlet does not pass objects through the pipeline.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PolicyName
+Name of the policy to be assigned.
+The PolicyName is simply the policy Identity without the policy scope i.e. the "Tag:" prefix.
+For example, a policy with the Identity Tag:ContosoPartnerCallHoldPolicy has a PolicyName equal to ContosoPartnerCallHoldPolicy.
+
+To revoke a per-user policy previously assigned to a user, set the PolicyName to a null value ($null).
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Tenant
+Globally unique identifier (GUID) of the tenant account whose Teams call hold policies are being queried.
+For example:
+
+-Tenant "38aad667-af54-4397-aaa7-e94c79ec2308"
+
+You can return your tenant ID by running this command:
+
+Get-CsTenant | Select-Object DisplayName, TenantID
+
+If you are using a tenant remote session of Windows PowerShell you do not have to include the Tenant parameter.
+Instead, the tenant ID will automatically be filled in for you based on your connection information.
+The Tenant parameter is primarily for use in a hybrid deployment.
+
+```yaml
+Type: System.Guid
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -WhatIf
+Shows what would happen if the cmdlet runs.
+The cmdlet is not run.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: wi
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### Microsoft.Rtc.Management.AD.UserIdParameter
+
+## OUTPUTS
+
+### System.Object
+## NOTES
+
+## RELATED LINKS
+
+[New-CsTeamsCallHoldPolicy](https://docs.microsoft.com/powershell/module/skype/new-csteamscallholdpolicy?view=skype-ps)

```

Comment:
For links to files in the same doc set (folder), use file-relative links (include the file name extension). Example: (New-CsTeamsCallHoldPolicy.md